File: Retos Abril/Java/Reto4.java
package RetosAbril;

public class Reto4 {
    public static void main(String[] args) {
        int contador = 5;

        do {
            if (contador > 0) {
                System.out.printf("""
                        __________
                       ( 00:00:0%d )
                        ----------
                       """, contador--);
            } else {
                System.out.printf("""
                        
                                    💥💥💥💥💥
                                  💥          💥
                                💥   00:00:0%d  💥
                                  💥          💥
                                    💥💥💥💥💥
                    """, contador--);
            }
        } while (contador >= 0);
    }
}

File: Retos Abril/Python/Reto_4.py
contador = 5

while contador >= 0:
    if contador > 0:
        print(f""" 
                __________
               ( 00:00:0{contador} )
                ----------
        """)
        contador -= 1
    else:
        print("""    
                    💥💥💥💥💥
                  💥           💥
                💥   00:00:0%d   💥
                  💥           💥
                    💥💥💥💥💥
        """)
        contador -= 1